tests/server/test_compilerservice.py

```diff
@@ -232,6 +243,16 @@ async def request_compile(env: data.Environment) -> uuid.UUID:
         print("request: ", results[0].id, env.id)
         return u1
 
+    async def get_compile(env: data.Environment, rid: uuid.UUID) -> data.Compile:
+        results = await data.Compile.get_by_remote_id(env.id, rid)
+        assert len(results) == 1
+        assert results[0].remote_id == rid
+        return results[0]
+
+    async def is_handled(env: data.Environment, rid: uuid.UUID) -> bool:
+        compile = await get_compile(env, rid)
+        return compile.handled
```

Comment:
We normally use `rid` to refer to a resource_id. This can be confusing.
```suggestion
    async def get_compile(env: data.Environment, remote_id: uuid.UUID) -> data.Compile:
        results = await data.Compile.get_by_remote_id(env.id, remote_id)
        assert len(results) == 1
        assert results[0].remote_id == remote_id
        return results[0]

    async def is_handled(env: data.Environment, remote_id: uuid.UUID) -> bool:
        compile = await get_compile(env, remote_id)
        return compile.handled
```